views/partials/header.html

```diff
@@ -32,13 +32,13 @@
 		</ul>
 		<ul class="o-header__nav-list">
```

Comment:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-subscriptions/amp-subscriptions.md suggests that maybe subscriptions-display is supported on elements including <div>s - can we stick a single only on the entire <ul> instead for simplicity?